e2e/integrations/integration.steps.ts

```diff
@@ -55,12 +54,20 @@ class IntegrationSteps {
     return page.selectAction(action);
   }
 
+
   @when(/^Camilla deletes the "([^"]*)" integration*$/)
   public deleteIntegration(integrationName: string): P<any> {
     const listComponent = new IntegrationsListComponent();
     return this.world.app.clickDeleteIntegration(integrationName, listComponent.rootElement());
   }
 
+  @when(/^she selects "([^"]*)" integration step$/)
+  public addStep (stepName: string): P<any> {
+    log.info(`Adding ${stepName} step to integration`);
+    const page = new IntegrationAddStepPage();
+    return page.addStep(stepName);
+  }
```

Comment:
@lvydra would you be with this small refactor? 

basically it replaces ` Then she selects "Filter" step`
with `And she selects "Filter" integration step` 

`Then` -> `When` because When is supposed to represent user action and `Then` is desired state (=assertion)
`step` -> `integration step` - easier to search in sources (try CTRL+F `step` - not very helpful compared to `integration step`). 
We already have actions named as `integration action`...